index.html

```diff
@@ -204,6 +204,21 @@ <h2 class="name">Web Page</h2>
       </div>
     </div>
   </div>
+  <h1 class="title">Contact me</h1>
+  <div class="footer">
+    <a class="social" href="https://github.com/BeJuanitox" target="_blank">
+      <i class="fa-brands fa-github itemSocial itemGh"></i>
```

Comment:
Try if this works: in all the icon tags

```suggestion
      <i class="fa-brands fa-github itemSocial itemGh" />
```